documentation/swagger.yml

```
@@ -2107,82 +2359,303 @@
           type: array
           description: Array with dates of application periods
           items:
-            $ref: "#/definitions/period_create"
+            $ref: "#/components/schemas/period_create"
         send_notifications:
           type: boolean
           default: false
           description: If notification sms, mail, push on impact should be send
         notification_date:
-          description: Date of notification (if not set and send_notification is true, it will be the current UTC date)
+          description: Date of notification (if not set and send_notification is true, it
+            will be the current UTC date)
           type: string
-          pattern: "^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}Z$"
-          example: "2018-08-03T12:19:13Z"
+          pattern: '^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$'
+          example: '2018-08-03T12:19:13Z'
         messages:
           type: array
           description: List with messages by channel
           items:
-            $ref: "#/definitions/disruption_impact_message"
+            $ref: "#/components/schemas/disruption_impact_message"
           uniqueItems: true
         objects:
           type: array
           description: List of impacted PT objects
+          items:
+            oneOf:
+              - $ref: "#/components/schemas/impact_object"
+              - $ref: "#/components/schemas/impact_line_section_object"
```

```suggestion
              - $ref: "#/components/schemas/impact_object_line_section"
```
Perhaps more readable